### DISCUSSION TITLE
guake-toggle not working

### DISCUSSION THREAD

**Discussion by mshzsh** [hyprwm/Hyprland#963] [Q&A] [answered]
`guake-toggle` not working after last hyprland update Of course, in addition to `guake-toggle` command, `guake` does not work properly after hyprland the latest update

**Comment by vaxerski** [comment]
git bisect hyprland and find the faulty commit then

**Comment by Dickby** [comment, marked as answer]
Try to set GUAKE_ENABLE_WAYLAND=1

**Reply by mshzsh** [reply]
It worked! Thanks
